server/recruitment.py

```diff
@@ -89,24 +98,78 @@ def edit_applicant_notes(applicant_user_id, text):
         applicant.notes = text
         return {'status': 'ok'}
 
+def get_user_list():
```

Comment:
This function is already present in admin.py, and it doesn't seem to be deleted from there in this PR. Make sure it's only present in one place.

Reply:
ah get_users yes

Reply:
done